test/test_websocket_api.py

```diff
@@ -0,0 +1,146 @@
+import os
+import time
+import requests
+from requests_toolbelt import MultipartEncoder
+from collections import OrderedDict
+import asyncio
+import websockets
+import pytest
+import json
+import uuid
+from pytest_check import check
+
+host = os.environ.get("BOPTEST_SERVER", "http://web")
```

Comment:
We may want to change BOPTEST_SERVER to just the domain name? I noticed the websockets.connect("ws://web") is currently hard coded and uses different protocol "ws" vs "http" 

BOPTEST_SERVER=api.dev.boptest.net

Reply:
how about BOPTEST_SERVER and BOPTEST_WS_SERVER so that we can config http versus https?